mongodbatlas/resource_mongodbatlas_cloud_provider_snapshot_backup_policy.go

```diff
@@ -304,9 +305,12 @@ func flattenPolicyItems(items []matlas.PolicyItem) []map[string]interface{} {
 }
 
 func flattenCloudProviderSnapshotBackupPolicy(d *schema.ResourceData, conn *matlas.Client, projectID, clusterName string) ([]map[string]interface{}, error) {
-	backupPolicy, _, err := conn.CloudProviderSnapshotBackupPolicies.Get(context.Background(), projectID, clusterName)
+	backupPolicy, res, err := conn.CloudProviderSnapshotBackupPolicies.Get(context.Background(), projectID, clusterName)
 	if err != nil {
-		if strings.Contains(fmt.Sprint(err), "BACKUP_CONFIG_NOT_FOUND") || strings.Contains(fmt.Sprint(err), "400") {
+		if res.StatusCode == http.StatusNotFound ||
+			strings.Contains(fmt.Sprint(err), "BACKUP_CONFIG_NOT_FOUND") ||
+			strings.Contains(fmt.Sprint(err), "Not Found") ||
```

Comment:
should we still have 400 too?  Just in case another case returns a 400 vs 404?

Reply:
I want to be sure that the validation validates only when the backup config is not found I checked the code documentation (https://docs.atlas.mongodb.com/reference/api/api-errors/) to find all related to backup_config to see the codes and I only found the 404 for this case.

Reply:
Good point - let me make sure that doc is updated and we aren't missing something.

Reply:
Thank you so much :)

Reply:
Per slack discussion resolving this as 404 should get us the error we want (for when no policy exists because CPS not enabled).  TY!